impl/src/main/java/impl/AnswerResourceImpl.java

```diff
@@ -45,7 +49,7 @@ public AnswerResourceImpl(AnswerDao answerDao, QuestionDao questionDao, DaoTrans
                 answer.setId(generatedKey.getKey());
                 return answer;
             }).onErrorResumeNext(throwable ->
-                Observable.error(new WebException(HttpResponseStatus.INTERNAL_SERVER_ERROR, throwable)));
+                error(new WebException(INTERNAL_SERVER_ERROR, throwable)));
```

Comment:
can we add logging or add a message to the webexception? that states that we could not add the answer to the question. It would be easier to understand the error when it occurs.

Reply:
If we log or change the error to return something, we may add a test that executes this line, because i think its not covered by tests right now.

Reply:
Yup, it's not covered, so I'll add a test.